modules/mw.cx.init.Translation.js

```
@@ -394,17 +394,17 @@ mw.cx.init.Translation.prototype.fetchDraftTranslation = function (
  */
 mw.cx.init.Translation.prototype.fetchDraftTranslationSuccess = function (draft, conflict) {
 	const admin_users = ['Doc James', 'Mr. Ibrahem 1'];
-	/*
+
 	if (admin_users.includes(mw.user.getName())) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)